managed/cmd/pmm-managed/main.go

```diff
@@ -36,6 +36,7 @@ import (
 	"sync"
 	"time"
 
+	_ "github.com/ClickHouse/clickhouse-go/v2"
```

Comment:
this was previously initialized in telemetry (since it was the only place we were using clickhouse), now that checks also use it - I guess it makes sense to bring it here.